instrumentation/opentelemetry_phoenix/mix.exs

```diff
@@ -62,18 +62,33 @@ defmodule OpentelemetryPhoenix.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:nimble_options, "~> 0.5 or ~> 1.0"},
-      {:opentelemetry_api, "~> 1.0"},
-      {:opentelemetry_telemetry, "~> 1.0"},
+      {:nimble_options, "~> 1.0"},
+      {:opentelemetry_api, "~> 1.3"},
+      {:opentelemetry_telemetry, "~> 1.1"},
       {:opentelemetry_process_propagator, "~> 0.3"},
-      {:opentelemetry_semantic_conventions, "~> 0.2"},
-      {:telemetry, "~> 0.4 or ~> 1.0"},
+      {:opentelemetry_semantic_conventions, "~> 1.27"},
+      {:otel_http, "~> 0.2"},
+      {:telemetry, "~> 1.0"},
       {:plug, ">= 1.11.0"},
       {:cowboy_telemetry, "~> 0.4", only: [:dev, :test]},
-      {:opentelemetry, "~> 1.0", only: [:dev, :test]},
-      {:opentelemetry_exporter, "~> 1.0", only: [:dev, :test]},
+      {:opentelemetry_exporter, "~> 1.7", only: [:dev, :test]},
+      {:opentelemetry, "~> 1.4", only: [:dev, :test]},
+      {:opentelemetry_bandit,
```

Comment:
Should we get these git dependencies resolved before merging?

Reply:
Yeah. It has to wait until those are published. Can't test or publish if the packages aren't on hex.